src/algebra/group/basic.lean

```diff
@@ -441,9 +443,15 @@ variables {G : Type u} [comm_group G]
 lemma mul_inv (a b : G) : (a * b)⁻¹ = a⁻¹ * b⁻¹ :=
 by rw [mul_inv_rev, mul_comm]
 
+@[to_additive]
+lemma div_eq_of_eq_mul' {a b c : G} (h : a = b * c) : a / b = c :=
+begin simp only [h, mul_comm, div_eq_mul_inv, mul_left_comm], rw [mul_left_comm], simp end
```

Comment:
```suggestion
by { simp only [h, mul_comm, div_eq_mul_inv, mul_left_comm], rw [mul_left_comm], simp }
```
Can you golf this into one `rw`? Currently it looks like the first `simp` is doing too much `mul_left_comm`.

Reply:
Done